<launch>
    <node pkg="imu_utils" type="imu_an" name="imu_an" output="screen">
        <param name="imu_topic" type="string" value= "/zed/imu/data_raw"/>
        <param name="imu_name" type="string" value= "zed"/>
        <param name="imu_bag" type="string" value= "/zed_imu.bag"/>
        <param name="data_save_path" type="string" value= "$(find imu_utils)/data/"/>
        <param name="start_time" type="int" value= "1"/>
        <param name="end_time" type="int" value= "110"/>
        <param name="max_cluster" type="int" value= "100"/>
    </node>
</launch>
